modules/zip_importer/includes/importer.inc

```diff
@@ -178,10 +178,29 @@ class ZipBatchImporter extends IslandoraBatchImporter {
 class ZipBatchImportObject extends IslandoraImportObject {
 
   protected $mods;
-  protected static $MARC2MODS;
```

Comment:
One needs to know when to diverge... The use of static not-quite-constants like this isn't very Drupal-y... The Drupal way would have the stylesheets and and identification methods for each defined in some kind of hook structure over which we iterate, but I'm not sure we would really want to go that far right now.